internal/manifests/querier.go

```diff
@@ -46,6 +45,12 @@ func NewQuerierStatefulSet(opts Options) *appsv1.StatefulSet {
 					},
 				},
 			},
+			{
+				Name: storageVolumeName,
+				VolumeSource: corev1.VolumeSource{
+					EmptyDir: &corev1.EmptyDirVolumeSource{},
+				},
+			},
```

Comment:
This is not required if we use the index-gateway as the queriers don't need the data directory anymore, same applies for the volumeMount.